recipes/demo_install.rb

```diff
@@ -167,7 +167,7 @@
 end
 
 cisco_ospf_vrf 'dark_blue default' do
-  auto_cost                45000
+  auto_cost                45_000
```

Comment:
Believe it or not, it's valid Ruby (and rubocop recommends it). See https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics